poem/src/middleware/csrf.rs

```diff
@@ -104,6 +104,12 @@ impl Csrf {
         Default::default()
     }
 
+    /// Sets the name of the csrf cookie. Default is `poem-csrf-token`.
+    #[must_use]
+    pub fn cookie_name(self, cookie_name: String) -> Self {
```

Comment:
I'd love receive something like `impl Into<String>` for better ergonomics

Reply:
Done ✅